### ISSUE TITLE
Not available in adapter list

### ISSUE THREAD

**Issue by AxelMeineke** [klein0r/ioBroker.gira-iot#53]
### The problem

Hi.
Tanks for the Adapter. But I can Not cind a stabile Version in the Adapter list.
"Please use the "adapter list" in ioBroker to install a stable version of this adapter."
What did I wrong? 

### Version of nodejs

Docker version

### Version of ioBroker js-controller

Docker version

### Version of Adapter

No version found

### Operating system running ioBroker

Linux

### Checklist of files to include below

- [X] iobroker.current.log (Please share the logs in debug mode)
- [ ] Contents of the JavaScript browser console (always include in cases of issues with the user interface)

### Additional information & file uploads

Not available

**Comment by klein0r** [comment]
You're right, the adapter is not in the stable adapter list right now (https://github.com/ioBroker/ioBroker.repositories/blob/master/sources-dist-stable.json)

Still waiting for some feedback here https://forum.iobroker.net/topic/58514/test-adapter-gira-iot/

**Comment by klein0r** [comment]
And the stable version of the web adapter needs to be increased: https://github.com/ioBroker/ioBroker.web/issues/279
